pkg/taskrun/taskrun.go

```diff
@@ -15,17 +15,39 @@
 package taskrun
 
 import (
+	"sort"
+
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
 var trGroupResource = schema.GroupVersionResource{Group: "tekton.dev", Resource: "taskruns"}
 
 type Run struct {
-	Name    string
-	Task    string
-	Retries int
+	Name           string
+	Task           string
+	Retries        int
+	StartTime      *metav1.Time
+	CompletionTime *metav1.Time
```

Comment:
May be we need to add CompletionTime https://github.com/tektoncd/cli/blob/c2f113546808911c915cdfd34789233f34a61ae6/pkg/cmd/pipelinerun/logs_test.go#L469

tb.TaskRunCompletionTime(time),